histories/elle/cas-ok.edn

```diff
@@ -0,0 +1,13 @@
+; Two processes read initial value nil
```

Comment:
Good and bad samples for CAS register are already exist, see `histories/knossos/cas-register/`.